biologging-sensor-data/src/components/overview/Table.css

```diff
@@ -6,4 +6,8 @@
 
 tr {
     cursor: pointer;
+}
+
+.hover-row:hover {
+    background-color: #262525 !important;
```

Comment:
Just out of curiosity, why is this style needed? When I commented it out, the behaviour was the same, but perhaps I just missed something

Reply:
I was added it when work before, forget to delete.